code/modules/core_implant/cruciform/machinery/cruciformforge.dm

```diff
@@ -0,0 +1,192 @@
+#define LOAD "load"
+#define WORK "work"
+#define DONE "done"
+
+/obj/machinery/neotheology/cruciformforge
+	name = "Cruciform Forge"
+	desc = "This unique piece of technology can be fed biomatter, plasteel, and gold in order to print more cruciforms. Only a select few NeoTheologian architects know how to create these machines. "
+	icon = 'icons/obj/neotheology_machinery.dmi'
```

Comment:
there was no specific folder for all machines?

Reply:
We have special folder for NT all machines.

Reply:
"Cruciform Forge" sprites need to be in neotheology_machinery.dmi, so they can be founded easily.

Reply:
I already hit the resolve conversation button.